src/service/auth-service.js

```diff
@@ -62,10 +62,15 @@ class AuthService {
             return false;
         }
 
-        return (
-            this._authConfig.publicOperations.includes(`v0/${operationName}`) ||
-            this._authConfig.publicOperations.includes(operationName)
-        );
+        const lowerCaseOperationName = operationName.toLowerCase();
+
+        return this._authConfig.publicOperations.some((publicOperation) => {
+            const lowerCasePublicOperation = publicOperation.toLowerCase();
```

Comment:
it might be better to transform strings into upper case because we have upper case in the db and also in the config